### ISSUE TITLE
Headless Mode

### ISSUE THREAD

**Issue by Iceball457** [mvdicarlo/postybirb#278]
tl;dr It would be very helpful if a headless or server only launch was possible from the command line

My personal computer is a dual boot system that isn't guaranteed to be on at all times. For running software full time (this and a few game servers, mostly) I have an Ubuntu server in the corner of my office.

However, as far as I can tell there's no way to launch this application at system boot on Linux. "Start PostyBirb on startup" is locked, running via systemd services fails because there's no X server active while there's no users logged in, running via systemd with Xfvd prevents seeing the GUI even after a user logs in, so it's useless in this case.

I can get the background parts of the application running by logging into my server via "ssh -X" and launching the PostyBirb GUI briefly (although I have to do this manually every time the server restarts, about twice a week). This won't remain feasible as my queue grows

```
Hardware acceleration disabled
[66563:0401/230543.212072:ERROR:ozone_platform_x11.cc(247)] Missing X server or $DISPLAY
[66563:0401/230543.212186:ERROR:env.cc(226)] The platform failed to initialize.  Exiting.
The futex facility returned an unexpected error code.
Aborted (core dumped)
```

**Comment by leaftail1880** [comment]
v4 now has headless mode! #279
